src/main/java/com/artipie/rpm/misc/StorageLock.java

```diff
@@ -49,12 +45,12 @@
     /**
      * Lock file name pattern.
      */
-    private static final String PTRN = "%s/lock-[a-z0-9-]{36}";
+    private static final String PTRN = "lock-[a-z0-9-]{36}";
 
     /**
      * Error message.
      */
-    private static final String ERROR = "Repository %s is already being updated, %d locks found.";
+    private static final String ERROR = "Repository '%s' is already being updated, %d locks found.";
```

Comment:
@olenagerasimova minor quibble here, let's say `...%d lock(s) found`, since it's possible there's just one lock.